tasks.py

```diff
@@ -46,16 +94,24 @@ def docker_compose(context, command, **kwargs):
     help={
         "force_rm": "Always remove intermediate containers",
         "cache": "Whether to use Docker's cache when building the image (defaults to enabled)",
+        "python-ver": "The version of Python to build the container with (default: 3.7)",
     }
 )
-def build(context, force_rm=False, cache=True):
+def build(context, force_rm=False, cache=True, python_ver=None):
```

Comment:
I don't particularly want to go back in the direction of adding per-command flags for parameters that need to apply globally. It's too easy to accidentally do something like `invoke build test --python_ver 3.6` and forget that this means that `build` will still default to `3.7`. We already now have two other ways to set the Python version (`invoke.yml` and environment variable) both of which avoid this particular pitfall.

Reply:
@jathanism can you chime in here since you you have the exact opposite ask and asked specifically for this to be added?

Reply:
Woof, I think there might be a little confusion here. As I understood it, it was that `python_ver` is literally only used during `build`, but I see now it's also been added to `vscode`. 

Even still, seeing as `--python_ver` is only an arg to `build`, for `test` it would be ignored, yes?

Reply:
I think there is some confusion here.  If someone runs the following

```shell
invoke build --python-ver 3.9
export INVOKE_NAUTOBOT_PYTHON_VER=3.6
invoke unittest
```

The tests will be run with python 3.9.  It would be fairly trivial to make it also test 3.9, I believe the trick would be to tag the docker container with the python version.  Then docker-compose would realize in the case above in order to run unittest it needs to build `nautobot:3.6-local` for example as only `nautobot:3.9-local` exists.  Today we are not including the python version in the tag so `nautobot:local` would exist after the build with 3.9 and be used for the unittest in the example above.  NOTE:  I am probably not using the right container names/tags here, this is an example.

Reply:
So in that case if you ran `build --python_ver 3.9` and then later ran `test` without any args, that the default `python_ver` would be used and cause test to rebuild?  If so that confirms the concerns of @glennmatthews and seems kind of not good.